compiler/infra/OMRCfg.hpp

```diff
@@ -312,6 +359,7 @@ class CFG
    TR::CFGNode *_pStart;
    TR::CFGNode *_pEnd;
    TR::Region _structureRegion;
+   TR::Region _internalRegion;
```

Comment:
This looks wrong - we shouldn't copy the region we should reference it... the _structureRegion looks wrong too. FYI @Leonardo2718.

Reply:
@andrewcraik I thought that the idea was for `OMRCfg` to own the region where edges are stored. This way, once `OMRCfg` is destroyed, the edges are freed.

Reply:
Right but if you are defaulting it to heap this will copy the heap which isn't the right thing to do...

Reply:
@efferifick never mind - my bad this is good - ignore me